data/mods/blazemod/blaze_override.json

```diff
@@ -94,6 +113,25 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "laser_rifle",
+        "name" : "mounted A7 laser rifle",
+        "symbol" : "t",
+        "color" : "magenta",
+        "broken_symbol" : "#",
+        "broken_color" : "magenta",
+        "damage_modifier" : 80,
+        "durability" : 400,
+        "range" : 12,
+        "item" : "laser_rifle",
+        "difficulty" : 8,
+        "location" : "on_roof",
+        "flags" : ["TURRET", "TOOL_NONE", "MANUAL", "UNMOUNT_ON_DAMAGE", "FOLDABLE", "NEEDS_BATTERY_MOUNT"],
```

Comment:
That `"NEEDS_BATTERY_MOUNT"` makes sense in blazemod where it forces the turret to be mounted on a turret mount, but in core game it would only be confusing.